stable/youtubedl-material/1.0.0/test_values.yaml

```diff
@@ -0,0 +1,53 @@
+# # Default values for YoutubeDL-Material.
+
+image:
+  repository: tzahi12345/youtubedl-material
+  pullPolicy: IfNotPresent
+  tag: 4.2
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+  supplementalGroups: [24, 44, 107]
```

Comment:
if it only runs as 1000 you need this:
```suggestion
# Configure the Security Context for the Pod
podSecurityContext:
  runAsNonRoot: true
  runAsUser: 1000
  runAsGroup: 1000
  fsGroup: 1000
  supplementalGroups: [24, 44, 107]
```

However:
I've seen some bad dockerfile design with collabora which required this weirdness:

```suggestion
# Configure the Security Context for the Pod
podSecurityContext:
  runAsNonRoot: false
  runAsUser: null
  runAsGroup: null
  fsGroup: null
```

Reply:
Ahh right, i had different values (568) on questions >,< 
I couldn't understand why tests are passing and i couldn't install it lol

Reply:
Ahh now I understand, I was hella confused.